docs/content/en/docs/intro-klt.md

```diff
@@ -7,7 +7,23 @@ cascade:
   github_subdir: "docs/content/en/docs"
   path_base_for_github_subdir: "/content/en/docs-dev"
 ---
-
+## What is Keptn?
+Keptn is an open-source platform that helps you automate the 
+deployment, monitoring, and remediation of your applications.
+Keptn is based on the GitOps methodology, which means that 
+it uses Git as the source of truth for your application deployments.
+This makes it easy to track changes to your applications and to roll back 
+to previous versions if necessary.
+
+## What is the Keptn Lifecycle Toolkit?
+
+The Keptn Lifecycle Toolkit (KLT) is a tool that helps you
```

Comment:
Not sure if we should mention KLT as a "Tool" when it says "Toolkit" in the name itself. KLT provides tools such as the metrics operator, lifecycle operator.

Reply:
The name could change but...  Really, we could just say "...KLT helps you automate..." and cut out "is a tool that"